### ISSUE TITLE
Update readme on how to accept the incoming urls / shared information 

### ISSUE THREAD

**Issue by matthijn** [langtube/expo-config-plugin-ios-share-extension#5]
Currently in the README.md there is no example on how to accept / handle the incoming data in the app. It would be a great addition to add that. I haven't been able to figure it out so far.

**Comment by timedtext** [comment]
Hey @matthijn , would you be interested in submitting a pull request for this issue?

**Comment by matthijn** [comment]
Yeah, that's no problem. As soon as I figure out how to read the data. 😄

**Comment by matthijn** [comment]
Can you nudge me in the right direction?

**Comment by matthijn** [comment]
Without adding code in the app, but only installing the extension, for me it doesn't seem to open the app. It does show the app in the available share screen. But pressing it, starts an animation like it's going to the app. But then it just stops (and I have to force close safari, since it's now a darker color). 

https://www.icloud.com/sharedalbum/#B0y5VaUrziocGj

**Comment by timedtext** [comment]
> Can you nudge me in the right direction?
Provide instructions in the README.md file on how to use patch-package to fix Xcode bugs.

**Comment by timedtext** [comment]
> and I have to force close safari, since it's now a darker color

I have also faced this problem. You need to set up the [scheme](https://github.com/timedtext/expo-config-plugin-ios-share-extension/blob/dd7235d9019b099e1cd5f6f2133fc66687bf2798/README.md?plain=1#L18) properly.

**Comment by matthijn** [comment]
Made PR available. https://github.com/timedtext/expo-config-plugin-ios-share-extension/pull/6

**Comment by timedtext** [comment]
Merged already. Thank you. @matthijn
